app/controllers/vacancies_controller.rb

```diff
@@ -19,13 +19,24 @@ class VacanciesController < ApplicationController
                 :sort_column,
                 :sort_order
 
+  # rubocop:disable Metrics/AbcSize
```

Comment:
I wonder if this is a good practice 🤔

Reply:
Rubocop is more of a guideline than a be all end all, so while not a perfect solution I don't have any problems with disabling and enabling specific linting as necessary

Reply:
I searched for a refactoring solution and ended up following the direction that was already taken in the codebase. i.e There are a number of places where rubocop was disabled.

Reply:
Have now refactored these. 👍